Hydrator/AbstractHydrator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace FR3D\LdapBundle\Hydrator;
+
+use FR3D\LdapBundle\Model\LdapUserInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * Provide a hydrator template for easy implementation.
+ */
+abstract class AbstractHydrator implements HydratorInterface
```

Comment:
I wrote this abstract for let the people extend from here and customize the createUser